en/docs/setup/prerequisites.md

```diff
@@ -18,3 +18,4 @@
      - Add Jetstack
     
          `helm repo add jetstack https://charts.jetstack.io`
+
```

Comment:
Don't we need to define the minimum helm and k8s versions required?

Reply:
Lets add this in separate PR